src/test/java/seedu/address/logic/parser/NoteCommandParserTest.java

```diff
@@ -31,8 +33,18 @@ public void parse_nameSpecified_success() {
     @Test
     public void parse_missingCompulsoryField_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, NoteCommand.MESSAGE_USAGE);
+
         // no parameters
         assertParseFailure(parser, NoteCommand.COMMAND_WORD, expectedMessage);
+
+        // no preamble
```

Comment:
this should be no command word instead of preamble, missing code coverage for non-empty preamble.

// non-empty preamble
assertParseFailure(parser, NoteCommand.COMMAND_WORD + " asdasdadcad" + NAME_DESC_AMY + PREFIX_NOTE + nonEmptyNote, expectedMessage);